### PR TITLE
add package_monitor

### PR DESCRIPTION
**Pull request by nickpell**
@kaapstorm

### CODE REVIEW

**Comment by nickpell** [comment]
https://github.com/yunojuno/django-package-monitor

File: settings.py [hunk 1]
```diff
@@ -413,8 +413,7 @@
     'custom.m4change',
 )
 
-INSTALLED_APPS = DEFAULT_APPS + HQ_APPS + ENIKSHAY_APPS
-
+INSTALLED_APPS = DEFAULT_APPS + HQ_APPS + ENIKSHAY_APPS + ('package_monitor',)
```
Comment (kaapstorm) [inline settings.py]: Should we maybe add `'package_monitor'` to `HQ_APPS` instead?

**Review by kaapstorm — APPROVED** [review]

File: requirements/requirements.txt [hunk 2]
```diff
@@ -103,3 +103,4 @@ contextlib2==0.5.4
 hiredis==0.2.0
 xlwt==1.3.0
 ddtrace==0.9.2
+git+git://github.com/nickpell/django-package-monitor@np/handle-no-semver
```
Comment (snopoke) [inline requirements/requirements.txt]: do we need this on prod?

Comment (nickpell) [reply requirements/requirements.txt]: Good point.  I've updated the PR so the app can only be used in dev environments.

File: localsettings.example.py [hunk 3]
```diff
@@ -201,6 +201,7 @@
 #    'testapps.test_pillowtop',
 #    'django_fsm', # Adds the ability to generate state diagrams for models using django-fsm
 #    'kombu.transport.django', # required for celery
+#    'package_monitor',
```
[the remark this answers is not in the document]
Comment (nickpell) [reply localsettings.example.py]: ignoring

File: settings.py [hunk 4]
```diff
@@ -2222,3 +2221,5 @@ def _determine_couch_databases(couch_databases):
             'NAME': 'custom.nic_compliance.password_validation.UsedPasswordValidator',
         }
     ]
+
+PACKAGE_MONITOR_REQUIREMENTS_FILE = os.path.join(FILEPATH, 'requirements', 'requirements.txt')
```
Comment (emord) [inline settings.py]: I think this could go in localsettings as well

File: urls.py [hunk 5]
```diff
@@ -172,6 +172,13 @@
     except ImportError:
         pass
 
+    try:
+        urlpatterns += [
```
Comment (emord) [inline urls.py]: You could also use `if settings.DEBUG`

Comment (nickpell) [reply urls.py]: there's one already (see above)

File: settings.py (hunk 4 above)
Comment (nickpell) [reply settings.py]: so the thing is this relies on FILEPATH which is defined in settings.  I didn't see any example of how you could access FILEPATH from localsettings

Comment (emord) [reply settings.py]: I think it should auto add it because earlier in the file we have `from localsettings import *`

Comment (nickpell) [reply settings.py]: (resolved)

**Review by kaapstorm — APPROVED** [review]

**Pull request merged**